curvesim/utils/__init__.py

```diff
@@ -1,16 +1,29 @@
 """Utlity functions for general usage in Curvesim."""
+__all__ = [
+    "get_env_var",
+    "get_pairs",
+    "dataclass",
+    "get_event_loop",
+    "cache",
+    "override",
+    "datetime",
```

Comment:
Perhaps I'm missing something, but it seems `.datetime` isn't imported below

Reply:
It's fine because it becomes part of the namespace due to how modules are loaded.